### ISSUE TITLE
CSS Grid for columns block

### ISSUE THREAD

**Issue by strarsis** [WordPress/gutenberg#21190]
**Is your feature request related to a problem? Please describe.**
Using CSS Grid for the `core/columns` block would make the styles much simpler and easier to customize. Currently, flexbox is used and complicated media queries with margins for emulating gaps between the columns (vertical gaps seem to be missing anyway).

**Describe the solution you'd like**
Styles using CSS Grid for the `core/columns` block base styles.

**Describe alternatives you've considered**
Adjusting existing flexbox styles is very cumbersome and error-prone, especially when it comes to readjusting media queries for switching the columns count.

**Comment by dnnsjsk** [comment]
I agree, especially since it is already being used in the Media/Text block. In my opinion the experience with it is much smoother!

**Comment by talldan** [comment]
There's lots of discussion about columns over on this issue - https://github.com/WordPress/gutenberg/issues/16370.

**Comment by mtias** [comment]
Closing as there is a CSS grid block now and we don't plan to refactor columns to use grid for the time being.
